site/content/docs/CICD/vercel.md

```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
 
 ## Blocking Vercel deployments
 
-If you have the `Block my deployment when a check fails` checkbox enabled, Checkly informs Vercel whether a Vercel deployment should be "blocked" based on the following rules, per check type:
+You can block your Vercel deployments if any of your checks fail by enabling the checkboxes highlighted in the
+screenshot below. 
 
-### API checks
-For **API checks** any failing check results in a blocked Vercel deployment. This is a simple, binary state: it either fails or doesn't.
+![blocking deployments](/docs/images/cicd/vercel/vercel_block.png)
 
-### Browser checks
+We have two flavours here...
```

Comment:
Giga nit: "flavours", "what it says on the tin" -> these colloquialisms might be immediately understandable for someone who speaks English at a comfortable level, but confusing for other folks.

Reply:
I think this is fine. It makes the docs readable